sample-app/redisearch-python-rest/server.py

```diff
@@ -1,44 +1,68 @@
-from flask import Flask, request, g
-import os
+from flask import Flask, request, json, jsonify
 from flask_cors import CORS
 
-from redisearch import *
-import redisearch.aggregation as aggregations
-import redisearch.reducers as reducers
-
 import redis
-
-server_port = os.getenv("SERVER_PORT", "8087");
-redis_url = os.getenv("REDIS_URL", "redis://localhost:6379");
-redis_index = os.getenv("REDIS_INDEX", "idx:movie");
+from os import environ
+from redis.commands.search import reducers
+from redis.commands.search.query import NumericFilter, Query
+import redis.commands.search.aggregation as aggregations
+
+redis_password = ""
+if environ.get('REDIS_SERVER') is not None:
+    redis_server = environ.get('REDIS_SERVER')
+    print("passed in redis server is " + redis_server)
+else:
+    redis_server = 'localhost'
+    print("no passed in redis server variable ")
+
+if environ.get('REDIS_PORT') is not None:
+    redis_port = int(environ.get('REDIS_PORT'))
+    print("passed in redis port is " + str(redis_port))
+else:
+    redis_port = 6379
+    print("no passed in redis port variable ")
+
+if environ.get('SERVER_PORT') is not None:
+    server_port = int(environ.get('SERVER_PORT'))
+    print("passed in redis port is " + str(server_port))
+else:
+    server_port = 8087
+    print("no passed in redis port variable ")
+
+if environ.get('REDIS_INDEX') is not None:
+    redis_index = environ.get('REDIS_INDEX')
+    print("passed in category file location " + redis_index)
+else:
+    redis_index = "idx:movie"
+    print("no passed in redis index using idx:movie")
+
+if environ.get('REDIS_PASSWORD') is not None:
+    redis_password = environ.get('REDIS_PASSWORD')
+    print("passed in redis password is " + redis_password)
+
+# conn = redis.StrictRedis(redis_server, redis_port)
+if redis_password is not None:
+    conn = redis.StrictRedis(redis_server, redis_port, password=redis_password, charset="utf-8", decode_responses=True)
```

Comment:
StrictRedis is Redis now... We support it still - but perhaps we can just use the Redis class?